graylog2-web-interface/src/components/common/LinkToNode.jsx

```diff
@@ -75,7 +75,10 @@ const LinkToNode = createReactClass({
     }
 
     return <i>Unknown Node</i>;
-  },
-});
+  }
+}
 
-export default LinkToNode;
+export default connect(
+  LinkToNode,
+  { nodeStore: NodesStore },
```

Comment:
we can extract the `nodes` directly here. this way `nodes` will be passed to the component instead of the `nodeStore` itself
```suggestion
  { nodeStore: NodesStore },
   ({ nodeStore, ...rest }) => ({
    ...rest,
    nodes: nodeStore.nodes,
  }),
```